File: 1044.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = lines.shift().split(' ');
var n1 = Number(n[0]);var n2 = Number(n[1]);
if (n1 % n2 === 0 || n2 % n1 === 0){
    console.log('Sao Multiplos');}
else{
    console.log('Nao sao Multiplos');}

File: 1044.cs
using System;
class URI {
    static void Main(string[] args) { 
        String [] x = Console.ReadLine().Split(' ');
        int n1 = int.Parse(x[0]);
        int n2 = int.Parse(x[1]);
        if(n1 % n2 == 0 || n2 % n1 == 0){
            Console.WriteLine("Sao Multiplos");
        }
        else{
            Console.WriteLine("Nao sao Multiplos");
        }
    }
}
